tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py

```diff
@@ -1058,6 +1057,9 @@ def _make_data_min_nonzero(self, xp, sp, axis):
                 if numpy.isinf(dm_data).all():
                     dm_data[0, 0] = 0
             else:
+                if axis < 0:
+                    axis += 2
+
```

Comment:
Is this needed?

Reply:
Yes, this is for the branch below:
https://github.com/cupy/cupy/blob/53db8f4047b252176401dc53657286b212fff0e4/tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py#L1067